docs/userguide/new_organization.rst

```diff
@@ -24,7 +24,7 @@ Otherwise, you can click on `Create an organization` in the Menu of the login sc
 .. note::
 
     Another possibility (though more technical) is to host you own Parsec server and create the organization on it using the Parsec CLI. You can then join that organization using an organization bootstrap URL.
-    You can read about this in the `the Hosting server <https://github.com/Scille/parsec-cloud/blob/master/docs/HOSTING.md>`_ part.
+    You can read about this in the `the Hosting server <https://github.com/Scille/parsec-cloud/blob/master/docs/adminguide/hosting/index.rst>`_ part.
```

Comment:
In RST you can use [`cross-referencing`](https://www.sphinx-doc.org/en/master/usage/referencing.html#ref-role).

The document we want to reference here already has a label

https://github.com/gurjot03/parsec-cloud/blob/e160ea69d5e966754dd680043d316e5ae83a4190/docs/adminguide/hosting/index.rst?plain=1#L5

So you can do:

```suggestion
    You can read about this in the :ref:`the Hosting server <_doc_adminguide_hosting>` part.
```

Reply:
please take a look now

Reply:
Yes, almost done https://github.com/Scille/parsec-cloud/pull/5656#discussion_r1375907014